src/scss/custom.scss

```
@@ -4,29 +4,33 @@
  * work well for content-centric websites.
  */
 
+@import url('https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap');
```

Stylelint (17.11.1)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap'" (import-notation)

(import-notation)